app/controllers/flights_controller.rb

```diff
@@ -3,30 +3,39 @@ def index
   end
```

Comment:
**Code quality - Refactor**

- Slim the` FlightsController` and reduce it's `flog` complexity.